src/hooks/use-toggle-false.tsx

```diff
@@ -0,0 +1,38 @@
+// 뭔가 별로네요.. ㅎㅎ
+import {useState} from "react";
+interface toggleValues {
+  initialText: string; // key값이 string이 아닐수 있으니깐 boolean이 나오는
+  boolValue: boolean;
+}
+
+type useToggleReturn = [toggleValues, () => toggleValues];
+// 네이밍 뭘로할까요..?
+function useToggleValues(initial: [string, string]): useToggleReturn {
+  const [toggle, setToggle] = useState<boolean>(true);
+
+  const toggleValue = {
+    initialText: toggle ? initial[0] : initial[1],
+    boolValue: toggle,
+  };
+
+  const onToggleState = (): toggleValues => {
+    setToggle(!toggle);
+    return toggleValue;
+  };
+  /*
+  const onToggleState = (): void => {
+    setToggle((prevToggle) => !prevToggle);
+  };
+  위 아래 같다고 볼수 있나요..?
+ const toggleState = {
+    comparison: toggle! ? initial[0] : initial[1],
+    value: toggle!,
+  }
+  이렇게 변할꺼 같은데 헷갈리네요*/
```

Comment:
값의 뒤쪽에 붙여주는 !(Non-null assertion operator)는 필요없을 것 같구요~
동작은 똑같을 것 같네요~
```suggestion
  /*
  const onToggleState = (): void => {
    setToggle((prevToggle) => !prevToggle);
  };
  위 아래 같다고 볼수 있나요..?
 const toggleState = {
    comparison: toggle ? initial[0] : initial[1],
    value: toggle,
  }
  이렇게 변할꺼 같은데 헷갈리네요*/
```

Reply:
이것도.. 실험용으로 만들던것인데 지웠습니다...!